packages/components/src/inputs/src/InputGroup.module.css

```diff
@@ -0,0 +1,92 @@
+.hop-InputGroup  {
+    --input-group-background: var(--hop-neutral-surface);
+    --input-group-border: var(--hop-neutral-border);
+
+    position: relative;
+
+    display: inline-flex;
+    gap: var(--hop-space-inline-sm);
+    align-items: center;
+
+    box-sizing: border-box;
+    inline-size: fit-content;
+    block-size: 2.5rem;
```

Comment:
missing a lot of CSS variable in this file

Reply:
yes, I'd put this as a variable.